tests/longevity/longevity_test.go

```diff
@@ -52,39 +51,39 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 	populateIntervals()
 	populateDisruptiveTriggers()
-	err := backup.UpdatePxBackupAdminSecret()
-	Expect(err).NotTo(HaveOccurred())
+	//err := backup.UpdatePxBackupAdminSecret()
+	//Expect(err).NotTo(HaveOccurred())
 })
 
 var _ = Describe("{Longevity}", func() {
 	contexts := make([]*scheduler.Context, 0)
 	var triggerLock sync.Mutex
 	triggerEventsChan := make(chan *EventRecord, 100)
 	triggerFunctions = map[string]func(*[]*scheduler.Context, *chan *EventRecord){
-		DeployApps:                      TriggerDeployNewApps,
-		RebootNode:                      TriggerRebootNodes,
-		RestartVolDriver:                TriggerRestartVolDriver,
-		CrashVolDriver:                  TriggerCrashVolDriver,
-		HAIncrease:                      TriggerHAIncrease,
-		HADecrease:                      TriggerHADecrease,
-		VolumeResize:                    TriggerVolumeResize,
-		EmailReporter:                   TriggerEmailReporter,
-		AppTaskDown:                     TriggerAppTaskDown,
-		CoreChecker:                     TriggerCoreChecker,
-		BackupAllApps:                   TriggerBackupApps,
-		BackupScheduleAll:               TriggerScheduledBackupAll,
-		BackupScheduleScale:             TriggerScheduledBackupScale,
-		BackupSpecificResource:          TriggerBackupSpecificResource,
-		BackupSpecificResourceOnCluster: TriggerBackupSpecificResourceOnCluster,
-		TestInspectBackup:               TriggerInspectBackup,
-		TestInspectRestore:              TriggerInspectRestore,
-		TestDeleteBackup:                TriggerDeleteBackup,
-		RestoreNamespace:                TriggerRestoreNamespace,
-		BackupUsingLabelOnCluster:       TriggerBackupByLabel,
-		BackupRestartPX:                 TriggerBackupRestartPX,
-		BackupRestartNode:               TriggerBackupRestartNode,
-		BackupDeleteBackupPod:           TriggerBackupDeleteBackupPod,
-		BackupScaleMongo:                TriggerBackupScaleMongo,
+		DeployApps:       TriggerDeployNewApps,
+		RebootNode:       TriggerRebootNodes,
+		RestartVolDriver: TriggerRestartVolDriver,
+		CrashVolDriver:   TriggerCrashVolDriver,
+		HAIncrease:       TriggerHAIncrease,
+		HADecrease:       TriggerHADecrease,
+		VolumeResize:     TriggerVolumeResize,
+		EmailReporter:    TriggerEmailReporter,
+		AppTaskDown:      TriggerAppTaskDown,
+		CoreChecker:      TriggerCoreChecker,
+		// BackupAllApps:                   TriggerBackupApps,
+		// BackupScheduleAll:               TriggerScheduledBackupAll,
+		// BackupScheduleScale:             TriggerScheduledBackupScale,
+		// BackupSpecificResource:          TriggerBackupSpecificResource,
+		// BackupSpecificResourceOnCluster: TriggerBackupSpecificResourceOnCluster,
+		// TestInspectBackup:               TriggerInspectBackup,
+		// TestInspectRestore:              TriggerInspectRestore,
+		// TestDeleteBackup:                TriggerDeleteBackup,
+		// RestoreNamespace:                TriggerRestoreNamespace,
```

Comment:
Same here

Reply:
Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint